src/test/java/codeu/model/store/persistence/PersistentStorageAgentTest.java

```diff
@@ -72,4 +79,13 @@ public void testWriteThroughMessage() {
     persistentStorageAgent.writeThrough(message);
     Mockito.verify(mockPersistentDataStore).writeThrough(message);
   }
+
+  @Test
+  public void testWriteThroughActivity() {
+    Activity activity =
+            new Activity(
```

Comment:
This should be indented only 4 spaces further in than the previous line.

Reply:
done